src/app/file-manager/file-modal/file-modal.component.html

```diff
@@ -0,0 +1,19 @@
+<h2 mat-dialog-title>Where do you want to download to?</h2>
+
+<mat-dialog-content>
+  <div class="flex-container">
+      <div class="browse-button">
+          <label for="directorypicker">Browse...</label>
+          <input type="file" id="directorypicker"  (change)="directoryChanged($event)" style="display: none;" webkitdirectory directory multiple required/>
+      </div>
+    <p class="browse-location">{{directory}}</p>
+    
+  </div>
+  <hr>
+  <mat-checkbox aria-label="Preserve folder structure"  (change)="selectionChanged($event)" color="primary" class="dialog-checkbox" disableRipple>Preserve local structure</mat-checkbox>
```

Comment:
I think the label 'Preserve local structure' is confusing as the structure being preserved is the remote (cloud) structure not the local one. I think 'Preserve folder structure' is better for the checkbox label.